ui/app/components/secret-engine/list.hbs

```diff
@@ -0,0 +1,116 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Toolbar>
+  <ToolbarFilters>
+    <SearchSelect
+      @id="filter-by-engine-type"
+      @options={{this.secretEngineArrayByType}}
+      @selectLimit="1"
+      @disallowNewItems={{true}}
+      @fallbackComponent="input-search"
+      @onChange={{this.filterEngineType}}
+      @placeholder={{"Filter by engine type"}}
```

Comment:
If it's a string you don't need the curly brackets. Curly brackets are for vars, ex: @placeholder={{this.placeholder}}
```suggestion
      @placeholder="Filter by engine type"
```